user-management/src/test/java/de/conciso/keycloak/user/mgmt/ListenerITest.java

```diff
@@ -1,39 +1,34 @@
 package de.conciso.keycloak.user.mgmt;
 
-import com.auth0.jwt.JWT;
-import com.auth0.jwt.exceptions.JWTDecodeException;
-import com.auth0.jwt.interfaces.DecodedJWT;
 import com.github.dockerjava.api.exception.UnauthorizedException;
 import dasniko.testcontainers.keycloak.KeycloakContainer;
-import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.Test;
 import org.keycloak.admin.client.Keycloak;
 import org.keycloak.admin.client.KeycloakBuilder;
+import org.keycloak.representations.JsonWebToken;
 import org.keycloak.representations.idm.UserRepresentation;
+import org.keycloak.util.JsonSerialization;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testcontainers.containers.output.Slf4jLogConsumer;
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
+import java.io.IOException;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
+import java.util.Base64;
 
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @Testcontainers
-@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class ListenerITest {
     private static final Logger LOGGER = LoggerFactory.getLogger(ListenerITest.class);
     private static final String KEYCLOAK_VERSION = "23.0.4";
```

Comment:
I recommend avoiding hardcoding the Keycloak (KC) version in this section. Instead, pass the KC version to the test dynamically, for example, through a property like `System.getProperty("keycloak.version", "latest");`.

You can refer to this example for configuring failsafe plugin: https://github.com/sventorben/keycloak-restrict-client-auth/blob/main/pom.xml#L82

By default, use the KC version specified in the POM to ensure alignment with the version we are compiling the extension for.

Reply:
Is Fixed